### ISSUE TITLE
Visually compare multiple classification/regression methods

### ISSUE THREAD

**Issue by karlnapf** [shogun-toolbox/shogun#2718]
We aim to re-produce this plot:
http://scikit-learn.org/stable/_images/plot_classifier_comparison_001.png

This should be super simple within an ipython notebook (for example the one on classification).
Or within a new script. 

A similar thing for regression would be nice. 

Eternal fame is the reward as this will probably be added to the website ;)

**Comment by lisitsyn** [comment]
We will put your portrait in a postmodern style just to the left of this pic

**Comment by karlnapf** [comment]
Haha, WHAT? :)

**Comment by AnishShah** [comment]
Hi @karlnapf , I am new and I want to work on this. I want to know on which dataset should I make an IPython Notebook. ?

**Comment by abinashmeher999** [comment]
@karlnapf I am interested in this but I dont know if I will be suitable for this, since I dont know how to use any of those classifiers in shogun yet. At the same time when I am done with this, I would have learnt a lot, all the classifiers and a lot about shogun. I dont think there's any other issue that can teach me as much as this. I am all in for learning this but there are people who already know these and can do this in a very short time. I am apprehensive about taking this because already a PR of mine is pending which is further being delayed by my exams(which are still going on). Should I take this up?

@lisitsyn We will have to think of something to classify in Heiko's portrait. :)

**Comment by abinashmeher999** [comment]
@AnishShah they have used random sample generators from sklearn. Shogun might have a similar thing like this. Which I found [here](http://www.shogun-toolbox.org/doc/en/latest/classshogun_1_1CDataGenerator.html).

**Comment by lisitsyn** [comment]
Use this mask ;)

![699071](https://cloud.githubusercontent.com/assets/424629/6230701/bf9d824c-b6d2-11e4-9a0b-c561bb2789f5.jpeg)

**Comment by karlnapf** [comment]
hahaha ;)

**Comment by lisitsyn** [comment]
Task one - smooth this picture and use it as a distribution.

**Comment by karlnapf** [comment]
@AnishShah check the examples we have, all supervised methods we have are here:
http://www.shogun-toolbox.org/doc/en/latest/classshogun_1_1CMachine.html

**Comment by youssef-emad** [comment]
@karlnapf @lisitsyn I made this [notebook](https://github.com/youssef-emad/shogun_notebooks/blob/master/Classification.ipynb) for different classification models.

**Comment by karlnapf** [comment]
Hey @youssef-emad 
This is great, exactly what I had in mind. Thanks for that! Can you send a pull request? Then we can comment line by line!
